docs/dashboard.md

```diff
@@ -0,0 +1,61 @@
+# Dashboard
+
+Using dashboard you can monitor SC4SNMP and be sure that is healthy and working correctly.
+
+## Presetup Splunk and SC4SNMP
+
+1. [Create metrics indexes](gettingstarted/splunk-requirements.md#requirements-for-splunk-enterprise-or-enterprise-cloud) in Splunk.
+2. Enable metrics logging for your runtime:
+    * For K8S install [Splunk OpenTelemetry Collector for K8S](gettingstarted/sck-installation.md)
+    * For docker-compose use [Splunk logging driver for docker](dockercompose/9-splunk-logging.md)
+
+## Install dashboard
+
+1. In Splunk platform open **Search -> Dashboards**.
+2. Click on **Create New Dashboard** and make an empty dashboard. Be sure to choose Classic Dashboards.
+3. In the **Edit Dashboard** view, go to Source and replace the initial xml with the contents of [dashboard/dashboard.xml](https://github.com/splunk/splunk-connect-for-snmp/blob/main/dashboard/dashboard.xml) published in the SC4S repository.
+4. Saving your changes. Your dashboard is ready to use.
+
+
+## Metrics explanation
+
+### Polling dashboards
+
+For check that polling for your SNMP device working correctly first of all you need to look at **SNMP schedule of polling tasks** dahsboard.
+Using this chart you can understand when SC4SNMP scheduled polling for your SNMP device last time. If it regulary running is okay, otherwise you will see permanently zero scheduled tasks.
+
+After that we doublechecked that SC4SNMP scheduled polling tasks for your SNMP device we need to be sure that polling working correctly.
+For that look at another dashboard **SNMP polling status** and if everything is okay you will see only **succeeded** status of polling.
+If something going wrong you will see also another statuses (like on screenshot), then use [troubleshooting docs for that](bestpractices.md)
+
+*Note: if you set very big polling period like 2 hours, it's okay that you haven't found during this 2 hours new polling tasks*
```

Comment:
Generally for that note it depends, if something is changed in inventory the polling above 5 min will perform right after the walks, but if we only change frequency in profile and nothing in inventory, then the change will be visible after next walk will run. 
I thing it 's mentioned on the page with configuring polling/walks, so maybe we can copy the note from there also here.